docs/api/modules/Internationalization.md

```diff
@@ -0,0 +1,196 @@
+
+Below is an example of how you could include link specific data.
+
+Module Root:
+```
+module
+└── locale
+| └── en-US
+|     └── links
+|     |   └── production.json
+|     |   └── qa.json
+|     |   └── integration.json
+|     └── copy.json
+| └── es-MX.json
+|── src
+|   └── index.js
+└── package.json
+```
+
+Example `en-US/copy.json`:
+```JSON
+{
+  "loading": "Loading...",
+  "welcome": "welcome to my module"
+}
+```
+
+Example `en-US/links/production.json`:
+```JSON
+{
+  "example": "https://example.com/en-us/home"
+}
+```
+
+Example `en-US/links/qa.json`:
+```JSON
+{
+  "example": "https://qa.example.com/en-us/home"
+}
+```
+
+Example `en-US/links/integration.json`:
+```JSON
+{
+  "example": "localhost:8080/en-us/home"
+}
+```
+
+This will result in:
+```
+build
+└── 1.0.0
+    ├── <moduleName>.browser.js
+    ├── <moduleName>.legacy.browser.js
+    ├── <moduleName>.node.js
+    └── en-us
+    |    ├── <moduleName>.json
+    |    ├── integration.json
+    |    └── qa.json
+    └──es-MX
+        ├── <moduleName>.json
+        ├── integration.json
+        └── qa.json
+```
+
+Example `en-US/qa.json`:
+```JSON
+{
+  "loading": "Loading...",
+  "welcome": "welcome to my module",
+  "links": {
+    "example": "https://qa.example.com/en-us/home"
+  }
+}
+```
+
+### `ONE_CLIENT_LOCALE_FILENAME`
+
+One App uses the environment variable `ONE_CLIENT_LOCALE_FILENAME` to
+determine which language pack will be loaded. `ONE_CLIENT_LOCALE_FILENAME`
+must be one of `integration`, `qa`, or be `undefined` (for production).
+
+By default the it will be `undefined` unless `NODE_ENV === 'development'`
+which will result it in being set to `integration`
+
+Example:
+```bash
+ONE_CLIENT_LOCALE_FILENAME=integration
+```
+
+## Loading Language Packs
+
+One App uses [one-app-ducks intl duck](https://github.com/americanexpress/one-app-ducks#intl-duck) to manage
+the loading of each modules language pack.
+
+### Action Creators
+
+#### `loadLanguagePack`
+
+Used for fetching a module's language pack.
+
+Please see one-app-ducks [loadLanguagePack](https://github.com/americanexpress/one-app-ducks#loadlanguagepack) for information.
+
+#### `queryLanguagePack`
+
+The [Iguazu](https://github.com/americanexpress/iguazu) equivalent of [`loadLanguagePack`](#loadlanguagepack).
+
+Please see one-app-ducks [queryLanguagePack](https://github.com/americanexpress/one-app-ducks#querylanguagepack) for information.
+
+#### `updateLocale`
+
+Used to set the active locale for One App.
+
+Please see one-app-ducks [updateLocale](https://github.com/americanexpress/one-app-ducks#updatelocale) for information.
+
+#### `getLocalePack`
+
+Loads the locale of the requested country closest to the active locale. Used directly by One App.
+
+Please see one-app-ducks [getLocalePack](https://github.com/americanexpress/one-app-ducks#getlocalepack) for information.
+
+
+**📘 More Information**
+* Language packs compatible with [`react-intl`](https://github.com/formatjs/react-intl),
```

Comment:
period not comma IMO

Reply:
It is not clear what this sentence is trying to say. Maybe `To see an example of using [react-intl](...) within your module see...`